theSrc/scripts/Quadrants.js

```diff
@@ -120,6 +121,114 @@ function drawQuadrantColors (layout, config, ranges) {
         })
     }
 }
+function drawQuadrantTitles (layout, config, ranges, state) {
+    const { x_min, x_max, y_min, y_max } = ranges
+    if (!layout.annotations) layout.annotations = []
+    let index = layout.annotations.length
+    if (config.quadrantTopLeftTitle && config.yMidpoint < y_max &&
+        config.xMidpoint > x_min) {
+        const initial_offset = state.isStoredInState('quadrantTitle' + index)
+            ? state.getStored('quadrantTitle' + index)
+            : { ax: 0, ay: 0 }
+        layout.annotations.push({
+            text: config.quadrantTopLeftTitle,
+            xanchor: 'left',
+            xref: 'x',
+            ax: initial_offset.ax,
+            x: x_min,
+            xshift: 50,
```

Comment:
Should the offset be smaller so that the labels are closer to the corner? The positioning seems odd when the chart is small or when quadrants are small.

Reply:
Made offset smaller in [754a569](https://github.com/Displayr/rhtmlCombinedScatter/pull/79/commits/754a569504ff2a0c84d4bc99cc7dd686ea6a365c)